### DISCUSSION TITLE
Last release does not appear on Github Release

### DISCUSSION THREAD

**Discussion by fb64** [Kotlin/dataframe#632] [General]
I don't know it's wanted but the last release 0.13 does not appear on "Github Release"...

**Comment by Jolanrensen** [comment]
Haha, you were about an hour too early, I was still writing the notes :)

**Reply by fb64** [reply]
🎉  I asked because I didn't see the 0.13.0 version ...

**Reply by Jolanrensen** [reply]
Ah yes, well 0.13.0 was a bit of a released a bit by accident, hence the silence. But 0.13.1 was intentional :)  so enjoy and let us know if anything's wrong :)

**Reply by fb64** [reply]
Ok, no problem 👍
